src/pathplanning.org

```diff
@@ -0,0 +1,130 @@
+#+TITLE: Path Planning
+#+AUTHOR: Josh Ting
+#+EMAIL: josh.ting@gatech.edu
+#+REVEAL_THEME: black
+#+REVEAL_TRANS: linear
+#+REVEAL_SPEED: fast
+#+REVEAL_PLUGINS: (notes pdf)
+#+REVEAL_HLEVEL: 1
+#+OPTIONS: toc:nil timestamp:nil reveal_control:t num:nil reveal_history:t tags:nil author:nil
+
+# Export section for md
+* Path Planning {#t2016p}                                         :docs:
+
+* What is Path Planning
+- Our robots need to know how to get from a start point to an end point
+- RoboCup uses an RRT (Rapidly-Exploring Random Tree)
+- IGVC uses A*
+- STOx's Planner
+- Path network generation
+
+* RRT
+- Continuous search space
+- Randomly extend a space-filling tree throughout the search space until we connect the start and goal states
+
+* How the RRT works
+1. Start building our tree by placing a root node at the goal state
+2. Randomly select some coordinate in the position space
+3. Identify existing node in the tree that is nearest to that coordinate
+4. Add new node in tree branching from nearest node to random coordinate
+5. Repeat 2-4 until a node is created at our destination.
+    The series of connected nodes from root to destination forms the path that the robot will follow
+6. Smooth out path with overlaid Bézier curve
+
```

Comment:
I'm just going to switch to a live demo of the rrt at this point and then switch back the slides